Features/BanCommand.cs

```diff
@@ -0,0 +1,89 @@
+using DSharpPlus.Commands;
+using DSharpPlus.Entities;
+using DSharpPlus.Commands.ContextChecks;
+using ArtcordAdminBot.Features.Helpers;
+using System.IO;
+using System.Net.Http;
+using System.Text.Json.Serialization;
+
+namespace ArtcordAdminBot.Features
+{
+    public class BanCommand
+    {
+        private readonly IDatabaseService _databaseService;
+        private readonly HttpClient _httpClient;
+
+        public BanCommand(IDatabaseService databaseService)
+        {
+            _databaseService = databaseService;
+            _httpClient = new HttpClient();
+        }
+
+        [Command("ban")]
+        [RequirePermissions(DiscordPermissions.BanMembers)]
```

Comment:
Alright for now, we might later handle the permission with our custom permission system once we have one